<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База данных сотрудников ресторанной сети</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Рестораны</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="res_add">
          <property name="text">
           <string>Добавить ресторан</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="res_update">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="res_change">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Изменить адрес</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="res_remove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="res_search">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Поиск по адресу</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="6">
         <widget class="QListView" name="res_view">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Должности</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="pos_add">
          <property name="text">
           <string>Добавить должность</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pos_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Редактировать</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pos_remove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QTableView" name="pos_view"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pos_update">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Сотрудники</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QPushButton" name="staff_remove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Удалить запись</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="staff_add">
          <property name="text">
           <string>Добавить сотрудника</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="6">
         <widget class="QTableView" name="staff_view"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="staff_search">
          <property name="text">
           <string>Поиск по фамилии</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="staff_change">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Изменить запись</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="staff_update">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Статистика</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Выберите запрос:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="query_view"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionChangeAddress">
   <property name="text">
    <string>ChangeAddress</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
